data/korath ships.txt

```diff
@@ -955,6 +968,23 @@ ship "Model 16"
 	description "The Kor Mereti war drones are constructed in a modular fashion, which allows them to grow over time as more materials become available. This one is recently built, and relatively small."
 
 
+ship "Model 16" "Model 16 (Jump Miner)"
```

Comment:
The model 8's ship definition is only used in mining fleets, so indicating that it is a mining variant isn't really necessary.